fe1-web/__tests__/model/network/FromJson.ts

```diff
@@ -0,0 +1,496 @@
+/* eslint-disable */
+
+import {
+  ActionType, CloseRollCall,
+  CreateLao,
+  CreateMeeting, CreateRollCall,
+  ObjectType, OpenRollCall,
+  StateLao, StateMeeting,
+  UpdateLao, WitnessMessage
+} from '../../../Model/Network/Method/Message/data';
+import { ProtocolError } from '../../../Model/Network';
+import {Base64Data, Hash, PrivateKey, PublicKey, Signature, Timestamp} from "../../../Model/Objects";
+import { sign } from "tweetnacl";
+import { encodeBase64 } from "tweetnacl-util";
+
+const assert = require('assert');
+const assertChai = require('chai').assert;
+
+const STALE_TIMESTAMP = new Timestamp(1514761200);    // 1st january 2018
+const STANDARD_TIMESTAMP = new Timestamp(1609455600); // 1st january 2021
+const CLOSE_TIMESTAMP = new Timestamp(1609542000);    // 2nd january 2021
+const FUTURE_TIMESTAMP = new Timestamp(1735686000);   // 1st january 2025
+
+// FIXME use real keys from key pair once operational
+export const mockPublicKey = new PublicKey('xjHAz+d0udy1XfHp5qugskWJVEGZETN/8DV3+ccOFSs=');
+export const mockSecretKey = new PrivateKey('vx0b2hbxwPBQzfPu9NdlCcYmuFjhUFuIUDx6doHRCM7GMcDP53S53LVd8enmq6CyRYlUQZkRM3/wNXf5xw4VKw==');
+
+
+const _generateKeyPair = () => {
+  const pair = sign.keyPair();
+  const keys = { pubKey: encodeBase64(pair.publicKey), secKey: encodeBase64(pair.secretKey) };
+  return { pubKey: new PublicKey(keys.pubKey), secKey: new PrivateKey(keys.secKey) };
+};
+
+
+
+
+
+describe('=== fromJson object checks ===', function() {
+
+  const sampleKey1: PublicKey = _generateKeyPair().pubKey;
+  const sampleKey2: PublicKey = _generateKeyPair().pubKey;
+
+  const mockMessageId = Base64Data.encode('message_id');
+
+  const org = mockPublicKey;
+  const time = STANDARD_TIMESTAMP;
+  const name = 'PoP\'s team "LAO" or filling name';
+  const mockLaoId: Hash = Hash.fromStringArray(org.toString(), time.toString(), name);
+
+  let temp: any = {};
+
+  const sampleCreateLao: Partial<CreateLao> = {
```

Comment:
these samples are extremely precious. 
We might consider putting them in JSON files to share with the entire project.

We also discussed with @gnarula this possibility, as we could then re-use samples as part of both the documentation, validating the JSON schema and validating the sub-systems :)